packages/components/src/popover/index.js

```diff
@@ -407,22 +428,25 @@ const Popover = (
 			<div className="components-popover__content">{ children }</div>
 			{ hasArrow && (
 				<div
-					className="components-popover__arrow"
 					ref={ arrowRef }
+					className={ [
+						'components-popover__arrow',
+						`components-popover__arrow--${
+							placementData.split( '-' )[ 0 ]
+						}`,
```

Comment:
The [CSS coding guidelines](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/coding-guidelines.md#css) in this repo are "inspired by BEM" but not exactly BEM, so we want the modifier classes to be named `.is-top`, `.is-right`, etc.

Reply:
Super nit: why not just use string interpolation here?

Reply:
> The [CSS coding guidelines](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/coding-guidelines.md#css) in this repo are "inspired by BEM" but not exactly BEM, so we want the modifier classes to be named `.is-top`, `.is-right`, etc.

Done in https://github.com/WordPress/gutenberg/pull/42874/commits/097510867480c0187c5e536d16a0a0f11eecb731


> Super nit: why not just use string interpolation here?

I'm not sure I understand your suggestion, can you elaborate a bit more?

Reply:
I was suggesting:
```
className={`components-popover__arrow is-${ placementData.split( '-' )[ 0 ] }`}
```

Reply:
I see, thank you for the eplanation!

I personally tend to use the `[ ...classnames ].join(' ')` syntax out of habit — I find it tidier to read, and it allows me to easily add conditional classnames and add a filter to exclude them (ie.. `[ ...classnames ].filter(Boolean).join(' ')`)